### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by MashaLoban**

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -30,5 +33,42 @@ <h1>What do we offer</h1>
             </div>
         </div>
     </div>
+    </main>
+
+    <footer>
+        <div class="wrapper__footer">
+            <div class="footer__items">
```
Comment (ghost) [inline Triphouse-project/index.html:40]: Doesn't have sense to wrap ul into div.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -30,5 +33,42 @@ <h1>What do we offer</h1>
...
+                <div class="footer__item">
+                    <ul>
+                    <li class="footer__item_name">About</li>
+                    <li>How Triphouse works</li>
+                    <li>Careers</li>
+                    <li>Privacy</li>
+                    <li>Terms</li>
+                </ul>
+                </div>
+                <div class="footer__item">
```
Comment (ghost) [inline Triphouse-project/index.html]: Doesn't have sense to wrap ul into div.

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -30,5 +33,42 @@ <h1>What do we offer</h1>
...
+                    <li class="footer__item_name">Property types</li>
+                    <li>Guest houses</li>
+                    <li>Hotels</li>
+                    <li>Apartments</li>
+                    <li>Villas</li>
+                    <li>Holiday homes</li>
+                    <li>Hostels</li>
+                </ul>
+                </div>
+                <div class="footer__item">
```
Comment (ghost) [inline Triphouse-project/index.html]: Doesn't have sense to wrap ul into div.

File: Triphouse-project/index.html [hunk 4]
```diff
@@ -30,5 +33,42 @@ <h1>What do we offer</h1>
...
         </div>
     </div>
+    </main>
+
+    <footer>
+        <div class="wrapper__footer">
+            <div class="footer__items">
+                <div class="footer__item">
+                    <ul>
+                    <li class="footer__item_name">About</li>
```
Comment (ghost) [inline Triphouse-project/index.html]: True to use pseudo selector 'first-child'.

File: Triphouse-project/index.html [hunk 5]
```diff
@@ -30,5 +33,42 @@ <h1>What do we offer</h1>
...
+                    <li>How Triphouse works</li>
+                    <li>Careers</li>
+                    <li>Privacy</li>
+                    <li>Terms</li>
+                </ul>
+                </div>
+                <div class="footer__item">
+                <ul>
+                    <li class="footer__item_name">Property types</li>
```
Comment (ghost) [inline Triphouse-project/index.html]: True to use pseudo selector 'first-child'.

File: Triphouse-project/index.html [hunk 6]
```diff
@@ -30,5 +33,42 @@ <h1>What do we offer</h1>
...
+                    <li>Hotels</li>
+                    <li>Apartments</li>
+                    <li>Villas</li>
+                    <li>Holiday homes</li>
+                    <li>Hostels</li>
+                </ul>
+                </div>
+                <div class="footer__item">
+                <ul>
+                    <li class="footer__item_name">Support</li>
```
Comment (ghost) [inline Triphouse-project/index.html]: True to use pseudo selector 'first-child'.

File: Triphouse-project/style.css [hunk 7]
```diff
@@ -41,5 +45,43 @@
...
+    margin: 120px 103px auto 10%;
+    color: #FFFFFF;
+    size: 20px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 140%;
+}
+.footer__item li{
+    list-style-type: none;
```
Comment (ghost) [inline Triphouse-project/style.css:76]: 2021-04-12_13h41_09
Add margin for list item.

File: Triphouse-project/index.html (hunk 1 above)
Comment (MashaLoban) [reply Triphouse-project/index.html:40]: fixed

File: Triphouse-project/index.html (hunk 2 above)
Comment (MashaLoban) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/index.html (hunk 3 above)
Comment (MashaLoban) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/index.html (hunk 6 above)
Comment (MashaLoban) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/index.html (hunk 5 above)
Comment (MashaLoban) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/index.html (hunk 4 above)
Comment (MashaLoban) [reply Triphouse-project/index.html]: fixed

File: Triphouse-project/style.css (hunk 7 above)
Comment (MashaLoban) [reply Triphouse-project/style.css:76]: done. thanks)

**Pull request merged**